### DISCUSSION TITLE
Framerate judder in Fallout 4

### DISCUSSION THREAD

**Discussion by notquitereal** [LizardByte/community#140] [Sunshine]
Hi there! Apologies if this isn't the place to ask about _specific_ games, but Fallout 4 has been one of the few instances where Sunshine isn't running as expected so figured it might be representative of a broader problem. 

The game loads and plays fine, but there's a constant stutter/judder. There are very few visible compression artifacts on screen, and other games play perfectly, so I doubt it's an issue with streaming bandwidth. The game runs fine (albeit with much less fidelity) using steam streaming, and previously ran fine with Moonlight. 

Fallout 4 is a game that has some weird framerate stuff going on re: the speed of the simulation, and as such I've capped it manually using Nvidia's game specific settings. My suspicion is that something's going on with that and producing the stutter. Is this consistent with others' experiences? Is there a way I could remedy it?

Is there something specific about Sunshine's pipeline that would produce this, as opposed to Steam Streaming or Moonlight?

**Comment by smlcr** [comment]
I found answer. It is the game who caused this trouble. 
Solution: 

```
Navigate to your documents folder in your user folder. Then look for "My Games", then "Fallout4". 
In that folder there is a file called "Fallout4Prefs".
Look for these lines under the [Display] section and make sure they are set to these values:

bTopMostWindow=1
bMaximizeWindow=1
bBorderless=1
bFull Screen=0
iSize H=1080
iSize W=1920

Save the file, launch the game
```
My notes:
- It's not always helps, but this settings definetly needs to be setted up.
- If you have two monitors, Win+Shift+Arrow Left/Right helps to avoid this issue (but with settings at top)
- On the Nexus mods we have couple of mods that's possibly can fix this issue, but I didn't tested it. Mods like "Fallout 3 borderless fix" and etc

[a comment by an automated account is not shown]
